src/main/java/core/basesyntax/QuadrangularArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface QuadrangularArea {
+    default double getQuadrangularArea(double firstSide, double secondSide) {
```

Comment:
It's unusual to have default methods in interfaces that are meant to be purely abstract. Default methods are typically used to add new methods to interfaces in a backward-compatible way. If you need a common implementation, consider using an abstract class or providing a separate class that implements this interface and contains the common code.